### ISSUE TITLE
E404 Not found

### ISSUE THREAD

**Issue by Neur0z** [Proxwian/ts3tgbot#1]
Hi!
Today I tried to install this cool bot.
After installing nodejs I typed and running in terminal:
`npm install ts3tgbot`

and getting this:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/ts3tgbot - Not found
npm ERR! 404
npm ERR! 404  'ts3tgbot@latest' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can only contain URL-friendly characters
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-03-19T15_33_11_531Z-debug.log
```
Full log file:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'install', 'ts3tgbot' ]
2 info using npm@6.7.0
3 info using node@v11.12.0
4 verbose config Skipping project config: /root/.npmrc. (matches userconfig)
5 verbose npm-session f3e6501e52a5e0a0
6 silly install loadCurrentTree
7 silly install readLocalPackageData
8 http fetch GET 404 https://registry.npmjs.org/ts3tgbot 1495ms
9 http fetch GET 404 https://registry.npmjs.org/ts3tgbot 828ms
10 silly fetchPackageMetaData error for ts3tgbot@latest 404 Not Found - GET https://registry.npmjs.org/ts3tgbot - Not found
11 timing stage:rollbackFailedOptional Completed in 1ms
12 timing stage:runTopLevelLifecycles Completed in 2360ms
13 verbose stack Error: 404 Not Found - GET https://registry.npmjs.org/ts3tgbot - Not found
13 verbose stack     at res.buffer.catch.then.body (/usr/lib/node_modules/npm/node_modules/npm-registry-fetch/check-response.js:104:15)
13 verbose stack     at processTicksAndRejections (internal/process/next_tick.js:81:5)
14 verbose statusCode 404
15 verbose pkgid ts3tgbot@latest
16 verbose cwd /root
17 verbose Linux 4.15.18-9-pve
18 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "ts3tgbot"
19 verbose node v11.12.0
20 verbose npm  v6.7.0
21 error code E404
22 error 404 Not Found - GET https://registry.npmjs.org/ts3tgbot - Not found
23 error 404
24 error 404 'ts3tgbot@latest' is not in the npm registry.
25 error 404 Your package name is not valid, because
26 error 404 1. name can only contain URL-friendly characters
27 error 404 Note that you can also install from a
28 error 404 tarball, folder, http url, or git url.
29 verbose exit [ 1, true ]
```
What am I doing wrong?
thx

**Comment by Proxwian** [comment]
looks like my mistake in package forming :D will fix soon

**Comment by Neur0z** [comment]
Ok, thx )

**Comment by Proxwian** [comment]
Fixed with commit [87fcc445d2c7cd26fac9b5242474102d0d832a6e](https://github.com/Proxwian/ts3tgbot/commit/87fcc445d2c7cd26fac9b5242474102d0d832a6e)
